ui/widget.js

```diff
@@ -390,6 +408,55 @@ $.Widget.prototype = {
 		return this._setOptions({ disabled: true });
 	},
 
+	_classes: function( element, keys, extra, add, object ) {
+		var full = [],
+			that = this;
+
+		object = object || this.options.classes;
+
+		function processClassString( classes, checkOption ) {
```

Comment:
It needs access to all of the variables inside `_classes` so it would be awkward to separate it